src/test/java/com/verisign/storm/metrics/GraphiteMetricsConsumerTest.java

```diff
@@ -184,17 +210,16 @@ public void shouldReadDataAndSendLong(Collection<DataPoint> dataPoints) {
     return new Object[][] { new Object[] { dpList } };
   }
 
-  @Test(dependsOnMethods = { "shouldReadConfigAndContext" }, dataProvider = "generateDataPointsFloat")
+  @Test(dependsOnMethods = { "shouldInitializeGraphiteReporter" }, dataProvider = "generateDataPointsFloat")
```

Comment:
Same here -- `shouldReadDataAndSendFloat` only applies to the Graphite backend, not Kafka.  Do you think we should add the equivalent tests for Kafka, too?